internal/pkg/dashboard/components/talosinfo.go

```diff
@@ -158,8 +185,28 @@ func (widget *TalosInfo) redraw() {
 				Name:  "MACHINES",
 				Value: data.numMachinesText,
 			},
+			{
+				Name:  "SECUREBOOT",
+				Value: data.secureBootState,
+			},
+			{
+				Name:  "STATE",
+				Value: data.statePartitionMountStatus,
+			},
+			{
+				Name:  "EPHEMERAL",
+				Value: data.ephemeralPartitionMountStatus,
+			},
 		},
 	}
 
 	widget.SetText(fields.String())
 }
+
+func mountStatus(encrypted bool, providers []string) string {
+	if !encrypted {
+		return "[green]OK[-]"
+	}
+
+	return fmt.Sprintf("[green]encrypted[-] (%s)", strings.Join(providers, ","))
```

Comment:
Maybe still keep the `OK`, sth like
```suggestion
	return fmt.Sprintf("[green]OK - encrypted[-] (%s)", strings.Join(providers, ","))
```

But it's up to you

Reply:
![image](https://github.com/siderolabs/talos/assets/18496730/cb2a5b23-99ad-46b6-b7ab-14bf2f4d612f)
fixed